apps/opik-frontend/vite.config.ts

```diff
@@ -37,8 +37,22 @@ export default defineConfig(({ mode }) => {
       sourcemap: true,
     },
     server: {
-      host: "0.0.0.0",
+      host: "0.0.0.0", // Allow access from network (enables mobile device testing)
```

Comment:
```
      // Allow network access by default for mobile testing
      // Set VITE_HOST=127.0.0.1 to restrict to localhost only
      host: env.VITE_HOST || "0.0.0.0",
```

Is possible, but would do in a separate task (As we already expose in 0.0.0.0)

Regarding mkcert for https locally, it's a nice to have, but also could be skipped for time being:

<img width="408" height="695" alt="image" src="https://github.com/user-attachments/assets/a8ea1b4b-e61a-4738-aa3a-ac3d5236e566" />

We could make it optional, than required, in a future ticket